app/view/Client/frmClient.cs

```diff
@@ -66,16 +74,22 @@ private void btnAddPatient_Click(object sender, EventArgs e)
         {
             using (frmClientPatientModal newClientPatientForm = new frmClientPatientModal())
             {
+                
                 newClientPatientForm.ShowDialog();
+                
             }
         }
 
         private void btnEditPatient_Click(object sender, EventArgs e)
         {
-            using (frmClientPatientModal newClientPatientForm = new frmClientPatientModal())
-            {
-                newClientPatientForm.ShowDialog();
-            }
+                if (dgvPatient.RowCount > 0)
+                {
+                    int petId = Convert.ToInt32(dgvPatient.SelectedRows[0].Cells["Id"].Value);
+                    MessageBox.Show("Are you sure you want to UPDATE ?");
```

Comment:
Change this messagebox to a YesNo confirmation button.

Sample
`if (MessageBox.Show("Do you want to update the selected record?", "Confirm to update", MessageBoxButtons.YesNo, MessageBoxIcon.Question) == DialogResult.Yes) {
          // code here
}`